src/styles/blocks/card.scss

```diff
@@ -0,0 +1,147 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  box-sizing: border-box;
```

Comment:
Remove to the * selector and remove it from all other other places.

Reply:
Still not fixed from the previous review
<img width="999" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/18043ec5-e65b-4bfb-9992-e80e02032712">